HPXMLtoOpenStudio/resources/schedule_files/occupancy-stochastic-residents-0.csv

```diff
@@ -0,0 +1,8761 @@
+occupants
+0
```

Comment:
What is the point of this file? The stochastic schedule generator won't even create such a file since we exit the measure if there are no residents.

Reply:
See comment above.